src/customRules/trailingCommaInterfaceRule.ts

```diff
@@ -0,0 +1,33 @@
+import * as ts from 'typescript';
+import * as Lint from 'tslint';
+
+export class Rule extends Lint.Rules.AbstractRule {
+
+     /* tslint:disable:object-literal-sort-keys */
+    public static metadata: Lint.IRuleMetadata = {
+        'ruleName': 'trailing-comma-interface',
+        'description': 'Enforce trailing commas in interface properties.',
+        'rationale': 'Our team uses traling commas in interfaces.',
```

Comment:
You can omit the end delimiter:

```ts
interface Foo {
  bar: string,
  qux: boolean
}
```

I suppose this rule should really be `interfaceLineEndDelimiter`, and enforce none/semicolon/comma/mixed.